go/ct/st/code_test.go

```diff
@@ -140,6 +140,57 @@ func TestCode_Printer(t *testing.T) {
 	}
 }
 
+func TestCode_OpCodesToString(t *testing.T) {
+	tests := map[string]struct {
+		code   *Code
+		start  int
+		length int
+		want   string
+	}{
+
+		"empty": {
+			code:   NewCode([]byte{}),
+			start:  0,
+			length: 4,
+			want:   "len(0)",
+		},
+		"complete": {
+			code:   NewCode([]byte{byte(ADD), byte(PUSH1), 0x0C, byte(PUSH2)}),
```

Comment:
nit: since `[]byte{byte(ADD), byte(PUSH1), 0x0C, byte(PUSH2)}` is repeated for almost all cases, consider having a `baseCode` variable.